assets/tests/components/routeVariantName.test.tsx

```diff
@@ -88,33 +91,35 @@ describe("RouteVariantName", () => {
       viaVariant: "_",
     }
 
-    const wrapper = shallow(<RouteVariantName vehicle={testVehicle} />)
+    const result = render(<RouteVariantName vehicle={testVehicle} />)
 
-    expect(wrapper.text()).toEqual("39_")
+    expect(result.getByTestId("variant-name")).toHaveTextContent("39_")
   })
+})
```

Comment:
It looks like this closes the `describe(...)` call and the remaining tests are moved one level of indentation up, outside of a `describe`. Was this intentional?